conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -88,6 +88,8 @@ TEST_F(
     metricsTimestampProperties.pNext = nullptr;
     metricGroupProp.pNext = &metricsTimestampProperties;
 
+    ASSERT_GT(differentDomainsMetricGroupHandles.size(), 0u)
```

Comment:
beyond the scope of this PR: It seems we need a new Metrics capability to tag tests requiring more than one metric source.  For instance, client platforms of Xe1 (e.g. DG2) only have OA metrics. So this will always assert.

Reply:
Resolved, thank you.